pytest_selenium/pytest_selenium.py

```diff
@@ -136,29 +131,50 @@ def pytest_runtest_makereport(__multicall__, item, call):
             if hasattr(cloud, driver_name.lower()) and driver.session_id:
                 provider = getattr(cloud, driver_name.lower())
                 extra_summary.append('%s Job: %s' % (
-                    provider.name, provider.url(driver.session_id)))
+                    provider.name, provider.url(
+                        item.config, driver.session_id)))
                 if pytest_html is not None:
                     extra.append(pytest_html.extras.url(
-                        provider.url(driver.session_id),
+                        provider.url(item.config, driver.session_id),
                         '%s Job' % provider.name))
                     extra.append(pytest_html.extras.html(
                         provider.additional_html(driver.session_id)))
                 passed = report.passed or (report.failed and xfail)
-                provider.update_status(driver.session_id, passed)
+                provider.update_status(item.config, driver.session_id, passed)
         report.sections.append(('pytest-selenium', '\n'.join(extra_summary)))
         report.extra = extra
     return report
 
 
 def pytest_addoption(parser):
-    config = ConfigParser.ConfigParser(defaults={'baseurl': ''})
-    config.read('mozwebqa.cfg')
+    parser.addini('selenium_base_url', 'base url for selenium')
+
+    # browserstack configuration
+    parser.addini('browserstack_username',
+                  'browserstack username',
```

Comment:
Below (for sauce labs config) you include the `help=` argument name, which makes it clearer what the second argument is, but here you leave it out. I think for consistency and clarity you should add it here as well.